src/components/controller/controller.ts

```diff
@@ -1,9 +1,14 @@
 import AppLoader from './appLoader';
-import {Endpoints} from '../enums'
+import {Endpoints, Languages} from '../enums'
 import {Data} from '../intrerfaces'
 import { Callback } from '../dataTypes';
+// import LanguagesData from '../view/languages/languagesData';
+import { chosenLanguage } from '../view/languages/chosenLanguage';
+
 
 class AppController extends AppLoader {
+    // _chosenLanguage: string = 'en';
```

Comment:
Unused lines of code should be removed, not commented out

Reply:
Deleted unused lines